koku/api/report/queries.py

```diff
@@ -95,6 +95,16 @@ def check_view_filter_and_group_by_criteria(filter_set, group_by_set):
     return True
 
 
+def sanitize_tag(tag):
+    """Sanitize a tag by removing unwanted characters and URL-encoding it."""
+    characters_to_sanitize = r' "\'`;'
+    table = str.maketrans(characters_to_sanitize, "_" * len(characters_to_sanitize))
+    sanitized_tag = strip_prefix(tag, TAG_PREFIX).translate(table)
+    encoded_tag = str.encode(tag)
```

Comment:
I'm not sure why we are converting to bytes here, but it should be operating on `sanitized_tag`
```suggestion
    encoded_tag = str.encode(sanitized_tag)
```